netanim/animevent.h

```diff
@@ -45,14 +45,24 @@ class AnimEvent
     UPDATE_NODE_COUNTER_EVENT,
     CREATE_NODE_COUNTER_EVENT,
     IP_EVENT,
-    IPV6_EVENT
+    IPV6_EVENT,
+    END_EVENT // Dummy event
```

Comment:
What do you mean by END_EVENT? Probably the name should be changed... maybe NO_EVENT?